### PR TITLE
change the title to something great

### PR DESCRIPTION
**Pull request by colearendt**

### CODE REVIEW

**Comment by colearendt** [comment]
@philbowsher what do you think?

File: phils_notebook.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "R Notebook"
+title: "This is an amazing title. Really."
```
Comment (colearendt) [inline phils_notebook.Rmd:2]: This is not very title-like...?

File: phils_notebook.Rmd [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 output: html_notebook
```
Comment (colearendt) [inline phils_notebook.Rmd:3]: I'm glad you did not change this

File: phils_notebook.Rmd [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (colearendt) [inline phils_notebook.Rmd:1]:
```suggestion
---thisbreaksthings
```

**Review by colearendt — COMMENTED** [review]
Overarching comments. This looks good!

**Review by philbowsher — APPROVED** [review]
serenity now!

**Pull request merged**